lib/chef/resource/windows_security_policy.rb

```diff
@@ -80,13 +80,25 @@ class WindowsSecurityPolicy < Chef::Resource
       property :secvalue, String, required: true,
       description: "Policy value to be set for policy name."
 
+      load_current_value do |desired|
+        secopt_values = load_secopts_state
+        output = powershell_out(secopt_values)
+        if output.stdout.empty?
+          current_value_does_not_exist!
+        else
+          state = Chef::JSONCompat.from_json(output.stdout)
+        end
```

Comment:
* There could be whitespace in this output that would break the `.empty?` tests.
* It'd be a small bit easier to read if this were changed into more a guard-clause style.

```suggestion
        output = powershell_out(secopt_values).stdout.strip
        current_value_does_not_exist! if output.empty?
          
        state = Chef::JSONCompat.from_json(output)
```